docs/plugins/woodpecker-plugins/plugins.json

```diff
@@ -93,6 +93,11 @@
       "name": "Gitea Release",
       "docs": "https://codeberg.org/woodpecker-plugins/gitea-release/raw/branch/main/docs.md",
       "verified": true
+    },
```

Comment:
https://gitnet.fr/deblan/woodpecker-email/raw/branch/develop/DOCS.md

we need it's contet and not the rendered version ;)

Reply:
It was logical, though!